README.md

```diff
@@ -53,10 +78,10 @@ If you have a feature suggestion, please post your idea on our [User Voice](http
 
 ## Additional resources
 
-* [AAD DQ sample](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-diffquery)
-* [Working with Delta Query in Microsoft Graph](https://developer.microsoft.com/en-us/graph/docs/concepts/delta_query_overview)
-* [Microsoft Graph developer site](https://developer.microsoft.com/en-us/graph/)
-* [Call Microsoft Graph in an ASP.NET MVC app](https://developer.microsoft.com/en-us/graph/docs/platform/aspnetmvc)
-* [MSAL Sample](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet)
+- [AAD DQ sample](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-diffquery)
+- [Working with Delta Query in Microsoft Graph](https://developer.microsoft.com/en-us/graph/docs/concepts/delta_query_overview)
+- [Microsoft Graph developer site](https://developer.microsoft.com/en-us/graph/)
+- [Call Microsoft Graph in an ASP.NET MVC app](https://developer.microsoft.com/en-us/graph/docs/platform/aspnetmvc)
+- [MSAL Sample](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet)
```

Comment:
Could you please rather propose MSAL.NET, and the link be https://aka.ms/msal-net ?